scripts/build/BUILD.gn

```diff
@@ -53,10 +53,10 @@ pw_python_package("build_examples") {
     "builders/host.py",
     "builders/imx.py",
     "builders/infineon.py",
-    "builders/k32w.py",
     "builders/mbed.py",
     "builders/mw320.py",
     "builders/nrf.py",
+    "builders/nxp/builder.py",
```

Comment:
this seems to be different from anything else. It seems beter to just have nxp.py for now.

Reply:
otherwise probably `nxp/parts.py` also needs adding